samples/app-anonymous-users/csharp/ClientApp/src/components/shareview.jsx

```diff
@@ -84,11 +112,195 @@ const ShareView = () => {
         }
     }
 
+    // Initiate facebook login.
+    const facebookLogin = () => {
+        fbAuthentication() // This method get a client-side token for Facebook
+            .then((result) => {
+                return getServerSideTokenFb(result.idToken); // This method get a face book user profile details.
+            })
+            .catch((error) => {
+                console.log(error);
+            });
+    }
+
+    // Get client side token for facebook.
+    const fbAuthentication = () => {
+        var facebookAppId = process.env.REACT_APP_FACEBOOK_APP_ID;
+        let redirectUri = window.location.origin + "/facebook-auth-end";
+
+        return new Promise((resolve, reject) => {
+            microsoftTeams.authentication.authenticate({
+                url: `https://www.facebook.com/v12.0/dialog/oauth?client_id=${facebookAppId}&redirect_uri=${redirectUri}&state=1234535`,
+                width: 600,
+                height: 535
+            })
+                .then((result) => {
+                    let data = localStorage.getItem(result);
+                    let tokenDetails = JSON.parse(data);
+                    localStorage.removeItem(result);
+                    resolve(tokenDetails);
+                })
+                .catch((reason) => {
+                    reject(reason);
+                })
+        });
+    }
+
+    // Get face book user profile details.
+    const getServerSideTokenFb = (clientSideToken) => {
```

Comment:
This function name doesn't seem to match what it actually does? Can we name it to something like get Facebook Profile Name

Reply:
Updated your feedback.